charts/shopware/values.yaml

```diff
@@ -176,6 +176,15 @@ store:
   #     business_event_handler_buffer:
   #       level: info
 
+  opensearch:
+    username: shopware-<APPLICATION-ID>
+    password: foo
```

Comment:
SecretRef like for the database. So we create the secret with the extra helm chart and reference it here. The operator will pull the secret and write it into the own shop secret. So we have the opensearch url later in shopware.

Reply:
Yes fair point. Let me update it.

Reply:
Done.